loopchain/peer/peer_outer_service.py

```diff
@@ -661,17 +661,6 @@ def VoteUnconfirmedBlock(self, request, context):
 
         return loopchain_pb2.CommonReply(response_code=message_code.Response.success, message="success")
 
-    def AnnounceNewLeader(self, request, context):
-        if not request.channel:
-            raise Exception("peer_outer_service:AnnounceNewLeader : Channel is not defined.")
-
-        logging.debug(f"AnnounceNewLeader({request.channel}): " + request.message)
-
-        channel_stub = StubCollection().channel_stubs[request.channel]
-        channel_stub.sync_task().reset_leader(request.new_leader_id)
-
-        return loopchain_pb2.CommonReply(response_code=message_code.Response.success, message="success")
-
```

Comment:
I think that would be better add `docstring` in announceNewLeader `deprecated` and nothing to do.
Then, refactoring with  removed radio station features.